### PR TITLE
add dev.paullee.scraterpreter.Scrapec

### PR DESCRIPTION
**Pull request by hillcrestpaul0719**
https://github.com/scraterpreter/scrapec

Thanks

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

**Comment by hillcrestpaul0719** [comment]
Sorry, I'm new to Flatpak and Flathub.

How can I fix
`stat: cannot stat 'builddir/*/share/app-info/xmls/dev.paullee.scraterpreter.Scrapec.xml.gz': No such file or directory` ?

**Comment by claudex** [comment]
I don't have a lot of experience with Flatpak, but I think you are missing metainfo and .desktop file (that are needed to generate dev.paullee.scraterpreter.Scrapec.xml.gz). https://docs.flatpak.org/en/latest/conventions.html#expected-standards

**Comment by hillcrestpaul0719** [comment]
Thanks

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

File: dev.paullee.scraterpreter.Scrapec.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    "path": "dev.paullee.scraterpreter.Scrapec.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "dev.paullee.scraterpreter.Scrapec.png"
+                }
+            ],
+            "builddir": true,
+            "post-install": [
+                "install -Dm644 dev.paullee.scraterpreter.Scrapec.png /app/share/icons/hicolor/128x128/apps/dev.paullee.scraterpreter.Scrapec.png",
```
Comment (bilelmoussaoui) [inline dev.paullee.scraterpreter.Scrapec.json:39]: You don't need an icon nor a desktop file for a console application

Comment (hillcrestpaul0719) [reply dev.paullee.scraterpreter.Scrapec.json:39]: Oh, okay. But then how do I set the thumbnail on the flathub?

Comment (bilelmoussaoui) [reply dev.paullee.scraterpreter.Scrapec.json:39]: The console applications are not displayed on the Flathub frontend yet

Comment (hillcrestpaul0719) [reply dev.paullee.scraterpreter.Scrapec.json:39]: Oh. So should I remove the icon or keep it in to keep the thumbnail for when console applications start being featured on the Flathub?

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrape

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

File: dev.paullee.scraterpreter.Scrapec.json (hunk 1 above)
Comment (hillcrestpaul0719) [reply dev.paullee.scraterpreter.Scrapec.json]: Anyhow I have deleted the icons and the desktop file. Thanks

**Comment by hillcrestpaul0719** [comment]
bot, build dev.paullee.scraterpreter.Scrapec

File: dev.paullee.scraterpreter.Scrapec.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline dev.paullee.scraterpreter.Scrapec.metainfo.xml:1]: Ideally you should maintain in the upstream repository.

**Pull request closed without merging**